### PR TITLE
Update package to include driect invoke events

### PR DESCRIPTION
**Pull request by RobinRobinG**

### CODE REVIEW

File: src/parse-apigw-http-event.js [hunk 1]
```diff
@@ -14,7 +17,7 @@ module.exports = function parseApiGwHttpEvent(event){
   }
 
   return {
-    sourceType: 'api',
+    sourceType: ApiGateway,
```
Comment (jordanmatthews) [inline src/parse-apigw-http-event.js:33]: This feels like an odd way to use a constant... Instead import event and reference Event.ApiGateway

File: src/parse-dynamodb-new-image-event.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-    extractedRecords = event.Records.map(extractDynamoDBEntry).filter(entry => entry);
+  const extractDynamoDBEntry = record =>
+    record.dynamodb && record.dynamodb.NewImage;
+
+  const extractedRecords = event.Records.map(extractDynamoDBEntry).filter(
+    entry => entry
+  );
   return {
-    sourceType: 'dynamodb',
+    sourceType: DynamoDb,
```
Comment (jordanmatthews) [inline src/parse-dynamodb-new-image-event.js:12]: Same with using constants here

**Review by jordanmatthews — COMMENTED** [review]
Besides the use of the constants across all files, it seems good.

File: README.md [hunk 3]
```diff
@@ -2,17 +2,41 @@
...
+yarn add lambda-event-parse
+```
+
+Import the package
+
+```javascript
+const { parserEvent, Event } = require('lambda-event-parse');
+
+parserEvent(event);
+```
```
Comment (alpe88) [inline README.md:27]: should Event and event match?

File: README.md [hunk 4]
```diff
@@ -52,34 +77,35 @@ Let's take another example, an S3 Event:
...
-
 ```
 
 will be translated into:
 
 ```javascript
 {
-  sourceType: 's3',
+  sourceType: "s3",
   sourceEvent: originalEvent, // the complete, unchanged S3 event objet
```
Comment (alpe88) [inline README.md:98]: event object

File: README.md [hunk 5]
```diff
@@ -52,34 +77,35 @@ Let's take another example, an S3 Event:
...
-  records: [{
-    key: "HappyFace.jpg",
-    size: 1024
-  }] // array of events (with parameters as properties)
+  records: [
+    {
+      "key": "HappyFace.jpg",
+      "size": 1024
+    }
+  ] // array of events (with parameters as properties)
```
Comment (alpe88) [inline README.md]: ];

??

**Review by alpe88 — APPROVED** [review]
Nice Job!

**Review by alpe88 — APPROVED** [review]

**Pull request merged**